packages/modules-runtime/server.js

```diff
@@ -15,7 +15,19 @@ makeInstallerOptions.fallback = function (id, parentId, error) {
   if (topLevelIdPattern.test(id)) {
     if (typeof Npm === "object" &&
         typeof Npm.require === "function") {
-      return Npm.require(id, error);
+      try {
+        return Npm.require(id, error);
```

Comment:
One risk of wrapping `Npm.require` in a `try` block is that the required module might throw an unrelated error while it's loading, which would get caught here and turned into a missing module error, hiding the true cause.

Another approach would be to modify the error thrown [here](https://github.com/meteor/meteor/blob/a770e9959fec4e499a8f8c01dac5c211808c72e3/tools/isobuild/bundler.js#L1926-L1928), in the implementation of `Npm.require`.

However, perhaps a simpler approach would be to move the `id.startsWith("meteor/")` check earlier, just inside the `if (topLevelIdPattern.test(id)) {` block. Reasoning: `Npm.require` doesn't have any logic for importing from `meteor/...` modules, so if the `id` starts with `meteor/`, `Npm.require` is almost certainly going to fail.